tests/helper/helper_generic.go

```diff
@@ -336,6 +336,21 @@ func GjsonMatcher(values []gjson.Result, expected []string) bool {
 	return matched == numVars
 }
 
+// GjsonExactMatcher validates if []results from gjson.GetMany match the expected values for each json path requested
+// Values is an array of results returned by the gjson.GetMany function
```

Comment:
Is adding the quotes as suggested below allowed in GoDoc? I suggested it because there didn't seem a better way to emphasize that it's a function argument we're talking about in the comment.
```suggestion
// "values" is an array of results returned by the gjson.GetMany function
```

Reply:
I'm not sure it is recommended to emphasize the arguments this way; the first example of this page (https://go.dev/blog/godoc) does not emphasize the `w` argument

Reply:
OK. Let's keep it as-is then. 👍🏾